egs/iam/v1/run.sh

```diff
@@ -44,7 +44,7 @@ fi
 if [ $stage -le 2 ]; then
   echo "$0: Preparing dictionary and lang..."
   local/prepare_dict.sh
-  utils/prepare_lang.sh --num-sil-states 4 --num-nonsil-states 8 --sil-prob 0.95 \
+  local/prepare_lang.sh --num-sil-states 4 --num-nonsil-states 8 --sil-prob 0.95 \
```

Comment:
What have you changed in prepare_lang.sh?

Reply:
it is calling local ->gen_topo file, to have different number of hmm states for punctuations.

Reply:
Do we actually need to do it for GMM systems too?
I imagine it does not matter much what topology we use for GMM models and we can use this special topology only for chain training.

Reply:
Jonathan, did experiments with changing topology for GMM systems and/or chain training. WER improved  if changing topology for GMM systems was changed. I took the modifications from Jonathan and added it, as it was improving the WER.  Maybe he can share more regarding the experiments.

Reply:
Different topologies for the chain training did not improve the WER. Only the changing the GMM topology improves the WER. From previous experiments the better GMM topology improved the chain training results from WER 5.88% to 5.32%. These number are from past results with LOB corpus included. I haven't run experiments to compare the results with the Wellington corpus included.